src/util/checkoutUtils.js

```diff
@@ -0,0 +1,104 @@
+import numeral from 'numeral';
+import myFTD from '@/graphql/query/myFTD.graphql';
+
+/** Format Transaction Data for Analtyics events
+* @param {Number} transactionId
+* @param {Array} loans
+* @param {Array} kivaCards
+* @param {Array} donations
+* @param {Object} totals
+ * @returns {Object}
+*/
+export function formatTransactionData(
```

Comment:
I created this file instead of adding to the mixin to make these easier to test. Tests will come a little later though.